src/Resources/config/resources.yml

```diff
@@ -0,0 +1,23 @@
+sylius_resource:
+  resources:
+    mango_order_comments_plugin.orderMessage:
+      classes:
+        model: MangoSylius\OrderCommentsPlugin\Entity\OrderMessage
+        form: MangoSylius\OrderCommentsPlugin\Form\Type\OrderMessageType
+
+swiftmailer:
```

Comment:
remove it, it does not belong to the plugin

Reply:
DONE